erpnext/education/doctype/student/student.py

```diff
@@ -68,7 +68,7 @@ def create_student_user(self):
 				'last_name': self.last_name,
 				'email': self.student_email_id,
 				'gender': self.gender,
-				'send_welcome_email': 1,
+				'send_welcome_email': self.send_welcome_email,
```

Comment:
`send_welcome_email` is property of student applicant so unaccesible here
Tests are failing too
https://github.com/frappe/erpnext/runs/2211410243?check_suite_focus=true#step:11:68